File: loop.py
print("1. UNCONDITIONAL / UNCONDITIONAL")
i = 1
while i < 5:
    print( "i = ", i)
    if i > 2:
        break
    i += 1
    
print("\n2. LABELED / UNLABELED")
i = 1
while i < 5:
    print( "\ni = ", i)
    j = 1
    while j < 5:
        print("j = ", j, ", " , end =" ")
        if j > 2:
            break
        j += 1
    i += 1

File: loop.php
<?php
echo "1. UNCONDITIONAL / UNCONDITIONAL\n";
for( $i = 1; $i < 5; $i++ ) 
{
    echo "i = $i\n";
    if( $i > 2 )
        break;
}

echo "\n2. LABELED / UNLABELED";
for( $i = 1; $i < 5; $i++ ) 
{
    echo "\ni = $i\n";
    for( $j = 1; $j < 5; $j++ ) 
    {
      echo "j = $j, ";
      if( $j > 2 )
        break;
    }
}

echo "\n*******";
for( $i = 1; $i < 5; $i++ ) 
{
    echo "\ni = $i\n";
    for( $j = 1; $j < 5; $j++ ) 
    {
      echo "j = $j, ";
      if( $j > 2 )
        break 2;
    }
}
?>
